pkg/events/check.go

```diff
@@ -291,7 +292,16 @@ func (ce *CheckEvent) Process(ctx context.Context) error {
 	ce.logger.Info().Msgf("adding %d apps to the queue", len(ce.affectedItems.Applications))
 	// Produce apps onto channel
 	for _, app := range ce.affectedItems.Applications {
-		ce.queueApp(app)
+		if len(ce.ctr.Config.AllowedNamespaces) > 0 {
+			ns := strings.Split(ce.ctr.Config.AllowedNamespaces[0], ",")
```

Comment:
What's going on here? In theory that slice should just be a list of namespaces, right? Is it not unmarshaling properly?

Reply:
Correct, I ran into this issue where the env var which is being such as `KUBECHECKS_ALLOWED_NAMESPACES: default,namespace-a` is converting this into a slice with single element, so something like `Allowed Namespace: '"httpdump-b,httpbin"'`. Hence, I am converting this comma separated list of strings to each element of slice.